HousingSearchListener/V1/UseCase/IndexTenureUseCase.cs

```diff
@@ -32,9 +33,22 @@ public async Task ProcessMessageAsync(EntityEventSns message)
                                          .ConfigureAwait(false);
             if (tenure is null) throw new EntityNotFoundException<TenureInformation>(message.EntityId);
 
-            // 2. Update the ES index
+            // 2. Get the asset for the tenure from the index
+            // TODO - The asset info should really be retrieved directly from the Asset Api and then a QueryableAsset rebuilt
```

Comment:
Is there a reason why we are not doing this now, with this PR?

Reply:
Because for this message processing we don't technically need to. We're not updating any asset info, only the tenure info on it, and we already have all the info we need for that.
Also to do so would require setting up a token in each environment in order to call the Asset Api and that is something only Humairaa can do at the moment and she's not here. This will have to get done when we start creating and updating assets but it's not necessary at the moment.